src/platform/testing/e2e/mockapi.js

```diff
@@ -70,10 +78,7 @@ function makeMockApiRouter(opts) {
     res.status(200).json(result);
   });
 
-  // Handle CORS preflight.
-  router.options('*', cors(corsOptions));
-
-  router.all('*', cors(corsOptions), (req, res) => {
```

Comment:
The `mocker-api` middleware also needed this CORS configuration, so I made it global on [line 131](https://github.com/department-of-veterans-affairs/vets-website/pull/12636/files#diff-f48c0ee0e00c3041f93dd06d3c4a7129R131).